### ISSUE TITLE
What I know of Fla formats

### ISSUE THREAD

**Issue by Cheemsandfriends** [ACrazyTown/AnimateAtlasGen#1]
From CS5 to Animate 2022, fla's are zip files but with a different file type.
They use XFLs, which are xmls essentially but with a specific way to assign shit, like the animation.json but different.

Now for the other ones I don't really know much, from what I searched, there are two types of old one, but they only differ on how they're assigned.
The thing that the both formats use is something called Compound File Binary. And from then, it's pure bytes.
Good luck with making a texture atlas from an fla tbh, I would begin with the xfl ones and later do the Macromedia to CS4 fla things.

I hope I can be of help even tho I'm pretty sure this would be almost impossible to pull off. But here's hoping

**Comment by ACrazyTown** [comment]
Yeah I'm not really sure if I'll finish this, the way FLA files store vector data and stuff is strange, I need to figure out how to parse them (and the weird ZIP flas). Thank you for your research though! Helps alot
